pkg/apis/compliance/v1alpha1/profile_types.go

```diff
@@ -35,12 +35,15 @@ type ProfilePayload struct {
 	// +optional
 	// +listType=atomic
 	Values []ProfileValue `json:"values,omitempty"`
+	// +optional
+	Version string `json:"version"`
```

Comment:
Note for reviewers - we need to make sure this is reasonable for future situations where a profile has multiple versions.

Right now, it looks like:

```
NAME        VERSION
ocp4-cis    1.4.0
pci-dss     3.2.1
```

But we will also likely support multiple versions of a single profile:

```
NAME        VERSION
ocp4-cis    1.4.0
pci-dss     3.2.1,4.0.0
```

Do we want this to be a comma separated list eventually? Or something else?

Reply:
should we use a string slices here

Reply:
Switched to using a slice of strings, which only supports a single version at the moment.